static/app/views/settings/organizationTeams/teamMembersRow.tsx

```diff
@@ -0,0 +1,146 @@
+import styled from '@emotion/styled';
+
+import Button from 'sentry/components/button';
+import IdBadge from 'sentry/components/idBadge';
+import {PanelItem} from 'sentry/components/panels';
+import RoleSelectControl from 'sentry/components/roleSelectControl';
+import {IconSubtract} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import space from 'sentry/styles/space';
+import {Member, Organization, User} from 'sentry/types';
+import {
+  hasOrgRoleOverwrite,
+  RoleOverwriteIcon,
+} from 'sentry/views/settings/organizationTeams/roleOverwriteWarning';
+
+const TeamMembersRow = (props: {
+  hasWriteAccess: boolean;
+  member: Member;
+  organization: Organization;
+  removeMember: (member: Member) => void;
+  updateMemberRole: (member: Member, newRole: string) => void;
+  user: User;
+}) => {
+  const {organization, member} = props;
+
+  return (
+    <TeamRolesPanelItem key={member.id}>
+      <div>
+        <IdBadge avatarSize={36} member={member} useLink orgId={organization.slug} />
+      </div>
+      <div>
+        <TeamRoleSelect {...props} />
+      </div>
+      <div>
+        <RemoveButton {...props} />
+      </div>
+    </TeamRolesPanelItem>
+  );
+};
+
+const TeamRoleSelect = (props: {
+  hasWriteAccess: boolean;
+  member: Member;
+  organization: Organization;
+  updateMemberRole: (member: Member, newRole: string) => void;
+}) => {
+  const {hasWriteAccess, organization, member, updateMemberRole} = props;
+  const {orgRoleList, teamRoleList, features} = organization;
+  if (!features.includes('team-roles')) {
+    return null;
+  }
+
+  const {orgRole: orgRoleId} = member;
```

Comment:
Do we need the full member or can we just pass the org role through?

Reply:
We use `member.teamRole` just 2 lines down. 

Doing it this way makes it slightly easier to separate between `*Role` vs `*RoleId`.